### ISSUE TITLE
Filter info tooltips have no text

### ISSUE THREAD

**Issue by BlueSlug** [inclusive-design/covid-data-monitor#19]
## Describe the bug
The info tooltips on each Accessibility Filter are missing any descriptive text or label.

### To reproduce
1. Go to https://wecount.inclusivedesign.ca/map
2. Note that the tooltip icons do not have any text associated with them. This can be observed in the "Accessibility" section of Dev Tools, or by using the site with a screen reader or other similar assistive technology

### Expected behavior
The info tooltips should provide information

## Screenshots
<img width="392" alt="2021-06-12 wecount a11y map tooltips" src="https://user-images.githubusercontent.com/547933/121830700-965f0300-cc93-11eb-8f7e-6fc0a0e3146e.png">

### Desktop
 - OS: Windows 10 v20H2
 - Browser: Chrome 91, Firefox 89
 - Version: 0.0.4

## Additional context or notes
N/A

**Comment by shubham-skr** [comment]
Hey, I am interested and want to work on this issue.

**Comment by cindyli** [comment]
@x-walker-x thanks for your interest in this ticket. We need to determine what texts to display in tooltips. This ticket needs more information to move forward.

**Comment by shubham-skr** [comment]
only the texts are missing or there's also some problem with the tooltip, because this issue is classified as a bug

**Comment by BlueSlug** [comment]
> only the texts are missing or there's also some problem with the tooltip, because this issue is classified as a bug

The issue is classified as a bug since that's currently the label that best describes what the problem is. The fix is going to be adding text, rather than a programming change.

**Comment by uttaraghodke** [comment]
**Accessible entrances:**  
The main entrance of the centre is accessible by a wheelchair or large scooter without a portable ramp. 
There are portable ramp available at the entrance. 
The centre has automatic doors.

**Accessible Washrooms**
The centre has wheelchair-accessible washrooms.

**Accessible Parking:**
The centre has accessible parking spaces.
The centre accessible parking meters or paying stations.

**Individual service:**
The centre provides sign language translation, second language translation, assistance for blind and visually impaired visitors
visitor request individual service on arriving.

**Wait accommodations:**
The centre accommodates visitors who are unable to queue. (e.g. priority service or registering a visitor’s spot in the queue while they wait elsewhere)

**Comment by uttaraghodke** [comment]
![image](https://user-images.githubusercontent.com/66276957/122788058-f0f3f280-d283-11eb-9c73-39eead355ca0.png)

**Comment by uttaraghodke** [comment]
![image](https://user-images.githubusercontent.com/66276957/122788305-2f89ad00-d284-11eb-86c3-885b72439579.png)
